site/source/docs/porting/pthreads.rst

```diff
@@ -4,6 +4,8 @@
 Pthreads support
 ==============================
 
+We are currently working on the shared memory support on COEP/COOP headers. It is a possibility that user's pthreads code will not work in deployed environment.For more information click `this <https://web.dev/coop-coep>`_
```

Comment:
This note is not correct. We (as in Emscripten) are not working on it. Instead, it's browsers that are shipping SharedArrayBuffer gated behind COOP/COEP, but then you also need to explain here what "COOP/COEP" abbreviations mean before linking out for details.